index.html

```diff
@@ -0,0 +1,13 @@
+<html>
+<head>
+	<title>Javascript Quiz</title>
+</head>
+<body>
+	<form>
+		<input type="text" id="height"/>
+		<input type="text" id="character"/>
```

Comment:
As a user, I had to guess which input was which. Be sure to keep users in mind, even on the smallest of interfaces